samples/glean/src/main/java/org/mozilla/samples/glean/GleanApplication.kt

```diff
@@ -7,13 +7,14 @@ package org.mozilla.samples.glean
 import android.app.Application
 import android.content.Intent
 import mozilla.components.lib.fetch.httpurlconnection.HttpURLConnectionClient
-import mozilla.components.service.experiments.Configuration as ExperimentsConfig
 import mozilla.components.service.glean.Glean
 import mozilla.components.service.glean.config.Configuration
 import mozilla.components.service.glean.net.ConceptFetchHttpUploader
-import mozilla.components.service.experiments.Experiments
+import mozilla.components.service.nimbus.Nimbus
```

Comment:
Would you kindly update the README file for this sample to replace `Integration of the Experiments library.` with the proper Nimbus wording? :D